lib/jsonapi/view.ex

```diff
@@ -91,6 +93,23 @@ defmodule JSONAPI.View do
 
       def fields, do: raise "Need to implement fields/0"
 
+      if Keyword.get(unquote(opts), :underscore_to_dash, false) do
```

Comment:
I think this would be better served as an application configuration since it would MOST likely be something they would use for the whole project and not a single view.